src/frontend/src/features/layouts/components/thread-view/components/assignees-widget/quick-assign-popover/_index.scss

```
@@ -0,0 +1,137 @@
+    border: 1px solid var(--c--contextuals--border--surface--primary);
+    border-radius: var(--c--globals--spacings--3xs, 4px);
+    box-shadow: 0 4px 16px 0 var(--c--contextuals--background--semantic--contextual--primary-hover);
+    overflow: hidden;
+    outline: none;
+}
+
+.quick-assign-popover__body {
+    display: flex;
+    flex-direction: column;
+    flex: 1;
+    // Without min-height:0 a flex item refuses to shrink below the
```

Stylelint (17.10.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)